qcsrc/defs.qc

```diff
@@ -379,6 +406,11 @@ float	CHAN_WEAPON		= 1;
 float	CHAN_VOICE		= 2;
 float	CHAN_ITEM		= 3;
 float	CHAN_BODY		= 4;
+#ifdef __QW__ // NetQuake doesn't support CHAN_NO_PHS_ADD. Set it to 0 so it will have no effect.
```

Comment:
Channel 0 actually is used, I think it serves as a fine fallback but you should revise this comment

Reply:
CHAN_NO_PHS_ADD only ever gets used as a modifier to an existing channel. I'll revise the message to make it clear why this will not affect netquake.